lib/lorekeeper/fast_logger.rb

```diff
@@ -19,6 +19,29 @@ def initialize(file)
       @level = DEBUG
       @iodevice = LogDevice.new(file)
       @file = file # We only keep this so we can inspect where we are sending the logs
+      @level_override = {}
+    end
+
+    def level
+      @level_override[Fiber.current] || @level
```

Comment:
No idea how this Fiber thing works or how expensive is to call this method

Reply:
umm, do people need this functionality? I am suspicious of anything that affects performance

Reply:
readme says

https://github.com/JordiPolo/lorekeeper/blob/03bb44dfd99c1614dff967d8b6dff5345f10605e/README.md?plain=1#L27

and this method is added in the stdlib's Logger so I added it